### ISSUE TITLE
README.md instructions for building/running don't work

### ISSUE THREAD

**Issue by tig** [microsoft/terminal#6457]
![image](https://user-images.githubusercontent.com/585482/84333067-e28cda00-ab4b-11ea-9437-b87df46ec902.png)

Running VS 2019 
![image](https://user-images.githubusercontent.com/585482/84333079-ec164200-ab4b-11ea-9233-5b65d26c12f3.png)

This is what I get:

![image](https://user-images.githubusercontent.com/585482/84333042-d012a080-ab4b-11ea-86b6-86e6f9c037bc.png)

What am I doing wrong?

**Comment by garciaolais** [comment]
Same issue 😞 

![image](https://user-images.githubusercontent.com/14133711/84340938-7f10a580-ab67-11ea-87ff-efdfe302f835.png)


![image](https://user-images.githubusercontent.com/14133711/84340590-b0d53c80-ab66-11ea-9e85-83eacdab2c64.png)

**Comment by skyline75489** [comment]
You need to choose `x64` instead of `Any CPU`. Try it and see if it works.

**Comment by garciaolais** [comment]
@skyline75489 thank you! 👍

**Comment by skyline75489** [comment]
Yeah we should probably add the "x64" part to the README. For programmers with only .NET background, "Any CPU" is way more natural than "x64". However, WT is a Win32 native project and can not be build with "Any CPU" configuration.

**Comment by zadjii-msft** [comment]
This is probably only a problem now that #6441 merged and there's an AnyCPU target that VS will default to, isn't it 😕

**Comment by DHowett** [comment]
Nah; there's been an AnyCPU target since the WPF Terminal control landed last year. :smile:
